tests/foreman/ui/test_hostunification.py

```diff
@@ -223,7 +224,11 @@ def test_positive_rename_foreman_host(self):
                 else:
                     self.fail('Host was not renamed')
                 self.assertIsNotNone(self.hosts.search(new_name))
+                if bz_bug_is_open(1495271):
```

Comment:
I think that one skip statement is redundant as we have redirection there already:
`self.hosts` -> `self.contenthost.`

Reply:
Redundant skip removed, but added new one between:
```
searched = self.hosts.search(vm.hostname)
```
and 
```
self.assertIsNotNone(self.hosts.search(new_name))
```
because it's the same view.